docs/faq/modeling.md

```diff
@@ -259,6 +259,26 @@ slots:
 
 See [dynamic enums](https://linkml.io/linkml/schemas/enums.html#dynamic-enums) for more details.
 
+### Are CURIEs used in schema definitions checked for expandability and resolution?  
+
+No, not at this time.  However, linkml_runtime does have methods to help you expand the CURIEs in your data
+using the prefixes in your model (see: linkml_runtime.utils.namespaces.py) into URIs and then checking them is 
```

Comment:
when we say checking do we mean:

- checking for syntactic validity according to the W3C URI specification
- link checking